README.md

```diff
@@ -157,42 +157,45 @@ The simplest way to get started making accurate rips is:
 
 ## Configuration file documentation
 
-The configuration file is stored according to the [XDG Base Directory Specification](
-http://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html)
-when possible.
+The configuration file is stored in  
+`$XDG_CONFIG_HOME/whipper/whipper.conf` or  
+`$HOME/.config/whipper/whipper.conf` if `$XDG_CONFIG_HOME` is undefined.
 
-It lives in `$XDG_CONFIG_HOME/whipper/whipper.conf` (or `$HOME/.config/whipper/whipper.conf`).
+See [XDG Base Directory
+Specification](http://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html)
+and [ConfigParser](https://docs.python.org/2/library/configparser.html).
 
-The configuration file follows python's [ConfigParser](https://docs.python.org/2/library/configparser.html) syntax.
+The configuration file consists of newline-deliniated `[sections]`
+containing `key = value` pairs. The sections `[main]` and
+`[musicbrainz]` are special config sections for options not respresented
+from the command line interface.  Sections beginning with `drive` are
+written by whipper; certain values should not be edited.
 
-The possible sections are:
+Example configuration demonstrating all `[main]` and `[musicbrainz]`
+options:
 
-- Main section: `[main]`
-  - `path_filter_fat`: whether to filter path components for FAT file systems
-  - `path_filter_special`: whether to filter path components for special characters
-  
-- MusicBrainz section: `[musicbrainz]`
-  - `server`: the MusicBrainz server to connect to, in `host:[port]` format. Defaults to `musicbrainz.org`.
-
-- Drive section: `[drive:IDENTIFIER]`, one for each configured drive. All these values are probed by whipper and should not be edited by hand.
-  - `defeats_cache`: whether this drive can defeat the audio cache
-  - `read_offset`: the read offset of the drive
+```Python
```

Comment:
Shouldn't this be `INI`? There's nothing Python specific about the configuration file, just standard INI file syntax, no?